### PR TITLE
Openjdk instead of oracle jdk

### PR DESCRIPTION
**Pull request by eivantsov**
Use official openjdk image

#### Changelog
Switch to OpenJDK, use official OpenJDK image as parent.

### CODE REVIEW

**Comment by skabashnyuk** [comment]
change log

**Comment by riuvshin** [comment]
@skabashnyuk @benoitf used `8u111-jre-alpine`

File: dockerfiles/codenvy/Dockerfile [hunk 1]
```diff
@@ -13,80 +13,19 @@
...
-           /opt/jdk/jre/lib/amd64/libgstreamer-lite.so \
-           /opt/jdk/jre/lib/amd64/libjavafx*.so \
-           /opt/jdk/jre/lib/amd64/libjfx*.so \
-           /opt/jdk/jre/lib/ext/jfxrt.jar \
-           /opt/jdk/jre/lib/ext/nashorn.jar \
-           /opt/jdk/jre/lib/oblique-fonts \
-           /opt/jdk/jre/lib/plugin.jar \
-           /tmp/* /var/cache/apk/* && \
-    echo 'hosts: files mdns4_minimal [NOTFOUND=return] dns mdns4' >> /etc/nsswitch.conf && \
+RUN echo "http://dl-4.alpinelinux.org/alpine/edge/community" >> /etc/apk/repositories && \
```
Comment (benoitf) [inline dockerfiles/codenvy/Dockerfile]: for which packages "edge" is required ?

Comment (benoitf) [inline dockerfiles/codenvy/Dockerfile]: also why do we need mc there ?

**Comment by riuvshin** [comment]
removed edge unzip and mc

**Pull request merged**